src/main/java/com/exactpro/th2/sailfish/utils/filter/NullFilter.java

```diff
@@ -19,23 +19,29 @@
 import com.exactpro.sf.aml.scriptutil.ExpressionResult;
 import com.exactpro.sf.aml.scriptutil.MvelException;
 import com.exactpro.th2.common.grpc.FilterOperation;
+import com.exactpro.th2.sailfish.utils.FilterSettings;
 import com.exactpro.th2.sailfish.utils.filter.util.FilterUtils;
+import org.jetbrains.annotations.NotNull;
+
+import java.util.Objects;
 
 public class NullFilter implements IOperationFilter {
     private static final String NOT_NULL = "*";
     private static final String NULL = "#";
     private final boolean acceptNull;
+    private final FilterSettings filterSettings;
```

Comment:
Please, remove reference to the whole settings object. You can use a boolean (or enum) parameter instead